packages/0x.js/test/0x.js_test.ts

```diff
@@ -1,28 +1,46 @@
+import { Deployer } from '@0xproject/deployer';
 import { BlockchainLifecycle, devConstants, web3Factory } from '@0xproject/dev-utils';
 import { BigNumber } from '@0xproject/utils';
 import * as chai from 'chai';
 import * as _ from 'lodash';
 import 'mocha';
+import * as path from 'path';
 import * as Sinon from 'sinon';
 
 import { ApprovalContractEventArgs, LogWithDecodedArgs, Order, TokenEvents, ZeroEx } from '../src';
 
+import { runMigrationsAsync } from './migrations/migrate';
 import { chaiSetup } from './utils/chai_setup';
 import { constants } from './utils/constants';
 import { TokenUtils } from './utils/token_utils';
 import { web3, web3Wrapper } from './utils/web3_wrapper';
 
+const artifactsDir = path.resolve('test', 'artifacts');
```

Comment:
Could we create a test util and import deployer from there as we do in other packages?